docs/book/component-gallery/model-registries/mlflow.md

```diff
@@ -0,0 +1,143 @@
+configured in your stack, the MLFlow Model Registry will also use the same
+configuration.
+{% endhint %}
+
+## How do you use it?
+
+
+### Built-in MLflow Model Registry step
+
+After registering the MLflow Model Registry component in your stack, you can
+use it in a pipeline by using the `mlflow_model_registry_step` which is a
+built-in step that is provided by the MLflow ZenML integration. This step
+automatically register the model that was produced by the previous step in the
+pipeline.
+
+```python
+# Pipeline run with MLflow model registry step
+mlflow_training_pipeline(
+    importer=loader_mnist(),
+    normalizer=normalizer(),
+    trainer=tf_trainer(params=TrainerParameters(epochs=5, lr=0.003)),
+    evaluator=tf_evaluator(),
+    model_register=mlflow_register_model_step(
+        params=MLFlowRegistryParameters(
+            name="Tensorflow-mnist-model",
+            description="A simple mnist model trained with zenml",
+            tags={"framework": "tensorflow", "dataset": "mnist"},
+            version_description=f"A run of the mlflow_training_pipeline with a learning rate of 0.0003",
+        )
+    ),
+).run()
+```
+
+### ZenML Command line interface (CLI)
+
+Sometimes adding a step to your pipeline is not the best option for you, as it
+will register the model in the MLflow Model Registry every time you run the
+pipeline. In this case, you can use the ZenML CLI to register your model
+manually. The CLI provides a command called `zenml model-registry models register-version`
+that you can use to register your model in the MLflow Model Registry.
+
+```shell
+zenml model-registry models register-version Tensorflow-model \
+    --model-uri="file:///.../mlruns/667102566783201219/3973eabc151c41e6ab98baeb20c5323b/artifacts/model" \
+    --tags key1 value1 --tags key2 value2 \
+    --description="A new version of the tensorflow model with accuracy 98.88%" \
+    --zenml-pipeline-name="mlflow_training_pipeline"
+```
+
+### List of available parameters
+
+To register a model version in the MLflow Model Registry, you need to provide
+list of parameters. when you use the built-in step, most of the parameters are
+automatically filled in for you. However, you can still override them if you
+want to. The following table shows the list of available parameters.
+
+* `name`: The name of the model. This is a required parameter.
+* `description`: A description of the registered model.
+* `tags`: A list of tags to associate with the registered model.
+* `model_uri`: The path to the model. This is a required parameter.
+* `version_description`: A description of the model version.
+* `version_tags`: A list of tags to associate with the model version.
+* `zenml_pipeline_name`: The name of the ZenML pipeline that produced the model.
+* `zenml_pipeline_run_id`: The run id of the ZenML pipeline that produced the
+model.
+* `zenml_pipeline_step_name`: The time when the ZenML pipeline that produced the
```

Comment:
Seems like the name and description of this are different things?

Reply:
sorry didn't understand what does this mean